test/e2e/tests/01-required-and-optional.js

```diff
@@ -132,9 +132,9 @@ module.exports = {
     expectNavigateAwayFrom(client, '/va-benefits/basic-information');
 
     // Financial disclosure page.
-    client.expect.element('input[name="understandsFinancialDisclosure"] + label').to.be.visible;
+    client.expect.element('input[name="understandsFinancialDisclosure-0"] + label').to.be.visible;
```

Comment:
This test should be checking `"understandsFinancialDisclosure-1"` so we can make sure the spouse, dependent, income, and expense fields work properly.